### PR TITLE
Update README with example output.

### PR DESCRIPTION
**Pull request by scottdangelo**
Include instructions to use the OpenScale Manage Application to
visualize bias, performance, and accuracy.

Closes: #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,13 +117,31 @@ bx resource service-instance <Watson_OpenScale_instance_name>
...
   Do **not** continue to the next cell until the code is finished running.
 
-# Sample Output
+### Sample Output
 
-![sample output](doc/source/images/dataMartOutput1.png)
+* Go to the instance of [Watson OpenScale](https://aiopenscale.cloud.ibm.com/aiopenscale/insights) that you created and click `Manage` on the menu and then `Launch Application`. Choose the `Insights` tab to get an overview of your monitored deployments, Accuracy alerts, and Fairness alerts.
 
-![sample output](doc/source/images/dataMartOutput2.png)
+![](doc/source/images/WOSinsights.png)
```
Comment (rhagarty) [inline README.md:125]: add tags to all images

File: README.md [hunk 2]
```diff
@@ -117,13 +117,31 @@ bx resource service-instance <Watson_OpenScale_instance_name>
...
+![](doc/source/images/WOSdetailedGraphs.png)
+
+* You can see from the image above that, at this time, our model is receiving 4.6 requests per minute, with an accuracy of 72%. We have 13% of our credit risk (and 87% of no risk) due the fact that this individual is age 18 to 25, and 28% credit risk (and 72% of no risk) due to the fact that this individual is female. This latter statistic id flagged as `bias`.
+
+* Now click `View details`.
+
+![](doc/source/images/WOStransactionDetailAge.png)
+
+* We can see that, for the `age` category, 87% of 18 to 25 year olds recieved `No Risk` compared to 82% of 26 to 75 year olds. This is not flagged as biased.
+
```
Comment (rhagarty) [inline README.md:137]: "received"

**Review by rhagarty — CHANGES_REQUESTED** [review]
typo and formatting issue

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md:125]: done

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md:137]: done

**Review by rhagarty — APPROVED** [review]

**Pull request merged**